examples/tilted_bottom_boundary_layer.jl

```diff
@@ -86,179 +65,173 @@ plot(grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz],
      xlabel = "Vertical spacing (m)",
      legend = nothing)
 
-
-# ## Setting up a buoyancy model in a tilted domain
+# ## Tilting the domain
 #
-# We set-up our domain in a way that the coordinates align with the tilted bottom. That means that,
-# from the coordinate's point of view, gravity is tilted by `θ_rad` radians, which we can simulate
-# by passing the `gravity_unit_vector` parameter to `Buoyancy()`:
-
-ĝ = [sin(params.θ_rad), 0, cos(params.θ_rad)]
-buoyancy = Buoyancy(model=BuoyancyTracer(), gravity_unit_vector=ĝ)
+# We use a domain that's tilted with respect to gravity by
 
+θ = 3 # degrees
```

Comment:
should we make this eg 10 degrees?
and perhaps make the movie plots tilted by `θ`?